styles.css

```diff
@@ -19,16 +21,24 @@ img {
 
 .grid {
   margin: 0 auto;
-  padding: 0 20px;
-  max-width: 980px;
+  padding: 35px 20px;
+  max-width: 80%;
+  display: grid;
+  grid-auto-flow: column;
+  grid-template-columns: 2fr 1fr;
```

Comment:
Try with `grid-template-columns: 3fr 1fr;`. You will get a result that is more like the layout provided.